packages/pwa-kit-react-sdk/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## v1.4.0-dev (Jan 06, 2022)
+-   Upgrade to React 17 [#278](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/278)
```

Comment:
I wonder if this is more correctly worded as "Add React 17 support". Looking at this from the eyes of a consumer of the sdk and the fact that React is a peer dependency that the consumer provides, it makes more sense.

Reply:
👍